src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    {
+        setFigureType("circle");
+    }
```

Comment:
first of all we do not need this filed, and the second thing is that static initialization blocks are not recommended to use